hack/make-rules/test-e2e-node.sh

```diff
@@ -57,21 +57,21 @@ fi
 # Setup the directory to copy test artifacts (logs, junit.xml, etc) from remote host to local host
 if [ ! -d "${artifacts}" ]; then
   echo "Creating artifacts directory at ${artifacts}"
-  mkdir -p ${artifacts}
+  mkdir -p "${artifacts}"
 fi
 echo "Test artifacts will be written to ${artifacts}"
 
 if [[ ${runtime} == "remote" ]] ; then
-  if [[ ! -z ${container_runtime_endpoint} ]] ; then
+  if [[ -n ${container_runtime_endpoint} ]] ; then
```

Comment:
Why did it change and what did it fix:
```
In ./hack/make-rules/test-e2e-node.sh line 65:
  if [[ ! -z ${container_runtime_endpoint} ]] ; then
        ^-- SC2236: Use -n instead of ! -z.
Ref: https://github.com/koalaman/shellcheck/wiki/SC2236
```